src/components/CheckForm/CheckForm.tsx

```diff
@@ -242,9 +252,33 @@ export const CheckForm = ({ check, disabled }: CheckFormProps) => {
               </FormLayout.Section>
               <FormLayout.Section label="Alerting" fields={[`alerts`, `alertSensitivity`]} status={status}>
                 <FeatureFlag name={FeatureName.AlertsPerCheck}>
-                  {({ isEnabled }) => (isEnabled ? <AlertsPerCheck onInitAlerts={handleInitAlerts} /> : null)}
+                  {({ isEnabled }) =>
+                    isEnabled ? (
+                      <>
+                        <TabsBar>
+                          <Tab
+                            label="Alerts-per-check"
```

Comment:
I know we've had various discussions on the naming and we seemed to have agreed to this name, however when discussing the documentation, the naming suggested has slightly changed to _"Per-check alerts"_. I believe that feels more natural for the docs, and probably for this tab itself. 
Should we adopt that naming? Happy to hear other thoughts on this.

Reply:
Changed it to `Per-check alerts` to be consistent with the docs https://github.com/grafana/website/pull/24518, but I'll change it if we reach consensus on something else 👍